camera/rpi-files/test_data_batcher.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+
+"""DataBatcher tests."""
+
+import pytest
+from data_batcher import DataBatcher
+
+
+def float_range(start, end):
+    """Create a list of floats from start to end, with step equal 1.
+
+    Args:
+        start: First list value
+        end: Last list value
+
+    Returns:
+        list of floats.
+    """
+    floats = []
+    for elem in range(start, end, 1):
+        floats.append(float(elem))
+    floats.append(float(end))
+    return floats
```

Comment:
not a big deal but python should get as simple as possible so I think `[float(x) for x in range(1, 10, 2)]` would be enough

Reply:
I didn't know about it, thanks!